standalone-e2e-test/src/test/java/AbstractRestAssuredBase.java

```diff
@@ -0,0 +1,37 @@
+import io.restassured.RestAssured;
+import org.junit.jupiter.api.BeforeAll;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+
+public class AbstractRestAssuredBase {
+    @BeforeAll
+    public static void init() {
+        String targetenv = System.getProperty("targetenv");
+        System.out.println("environment is " + targetenv);
```

Comment:
Please use logback as a logging library, follow this guide: https://www.baeldung.com/logback

As I remember RestAssured has build in logger, so don't worry about that one, just log some useful messages with logback.

Reply:
Done